core/src/main/java/org/elasticsearch/script/ClassPermission.java

```diff
@@ -89,6 +89,7 @@
             java.lang.Object.class.getName(),
             java.lang.Short.class.getName(),
             java.lang.String.class.getName(),
+            java.lang.StringBuffer.class.getName(),
```

Comment:
Are we going to leave this on since it doesn't get us anything here for this issue?